### PR TITLE
Adds missing find a housing counselor closing tags

### PR DESCRIPTION
**Pull request by anselmbradford**
Find a housing counselor pages throw a warning in IE about missing closing tags in some HTML elements on http://consumerfinance.gov/find-a-housing-counselor

screen shot 2018-04-04 at 12 04 24 pm

Looking at the templates it appeared they are missing a variety of closing `</div>`. `</span>`, and `</body>`.

## Changes

- Convert tabs to spaces.
- Fix template indenting.
- Adds closing `</div>`. `</span>`, and `</body>` where needed.

## Testing

1. Opening find a housing counselor templates, each nested level should have a corresponding closing tag.
1. /find-a-housing-counselor/ should be unchanged, other than there shouldn't be a warning in IE11.

### CODE REVIEW

File: cfgov/legacy/templates/hud/housing_counselor_pdf.html [hunk 1]
```diff
@@ -6,112 +6,134 @@
...
+                                        <div class="hud_hca_api_counselor">
+                                            <a href="{{ counselor.weburl }}" class="icon-link icon-link__external-link"><span class="icon-link_text">{{ counselor.nme }}</span></a>
+                                        </div>
+                                        <div class="hud_hca_api_counselor_address">
+                                            <span class="hud_hca_api_adr">
+                                                {{ counselor.adr1 }}
+                                                {% if counselor.adr2 and counselor.adr2 != ' ' %}
+                                                </br> {{ counselor.adr2 }}
+                                                {% endif %}
+                                            </span>
```
Comment (anselmbradford) [inline cfgov/legacy/templates/hud/housing_counselor_pdf.html]: This was missing.

File: cfgov/legacy/templates/hud/housing_counselor.html [hunk 2]
```diff
@@ -82,169 +82,178 @@ <h1>Find a housing counselor</h1>
...
-                                                    </tr>
-                                                    {% endwith %}
-                                                    {% endfor %}
-                                                </tbody>
-                                            </table><!-- end .hud_hca_api_results_listing -->
-                                        </div>
-                                        <div class="show-hide default-hidden hud_hca_api_print_hide hud_hca_api_pras">
-                                          <div>
-                                            <h3>Paperwork Reduction Act statement</h3>
+                                                            </span>
```
Comment (anselmbradford) [inline cfgov/legacy/templates/hud/housing_counselor.html]: This was missing.

File: cfgov/legacy/templates/hud/housing_counselor_pdf.html [hunk 3]
```diff
@@ -6,112 +6,134 @@
...
+                            </ol>
+                        </div>
+                    </div>
+                </div>
+            </article>
+        </section><!--/main_content-->
+    </div><!-- /wrapper -->
     <script type='text/javascript' src="{% static 'nemo/_/js/jquery-1.9.1.min.js' %}"></script>
     <script type='text/javascript' src="{% static 'nemo/_/js/functions.js' %}"></script>
+</body>
```
Comment (anselmbradford) [inline cfgov/legacy/templates/hud/housing_counselor_pdf.html]: This was missing.

File: cfgov/legacy/templates/hud/housing_counselor_pdf.html [hunk 4]
```diff
@@ -6,112 +6,134 @@
...
+                                        <h2 class="cfpb_visually_hide">Distance</h2>
+                                        <div class="hud_hca_api_distance">
+                                            <span class="hud_hca_api_the_distance"><span class="hud_hca_api_results_listing_title cfpb_hide hud_hca_api_print_show">Distance</span><span class="hud_hca_api_miles">{{ counselor.distance }}</span> miles</span>
+                                        </div>
+                                    </div>
+                                </li>
+                                {% endfor %}
+                            </ol>
+                        </div>
+                    </div>
```
Comment (anselmbradford) [inline cfgov/legacy/templates/hud/housing_counselor_pdf.html]: These two closing divs were missing.

**Comment by sebworks** [comment]
@anselmbradford, how do you feel HTML adding a linter in here as well ? It needs to run on Travis, otherwise we will never catch all of these.

**Comment by Scotchester** [comment]
@sebworks We're planning to look at this in the next couple days as a component of our work on automated accessibility testing! Basic validation can often prevent a lot of accessibility issues.

**Pull request merged**